andela/ah-fortem-frontend
# 166980306 Social authentication

#### Title

This PR allows users to sign in via social authentication
#### Description



Users should also be able to Login via their favourite social media platform

#### Type of change








#### How Has This Been Tested?
#### How Can This Be Manually Tested?
- To be able to test the feature, check out the branch `story/166980306-social-auth` and then `npm ci` followed by `npm run dev`.
- Go to Login or Sign up
- Click on any of the social media icons to sign in
![image](https://user-images.githubusercontent.com/44572764/61806205-f9491600-ae3f-11e9-9a76-edaf03e07cce.png)

#### Acceptance Criteria: 



#### Checklist:
- [x] Allows users to sign in via Facebook
- [x] Allows users to sign in via google
- [x] Allows users to sign in via twitter

#### PT stories
[#166980306](https://www.pivotaltracker.com/story/show/166980306)
